dbtcloud_blueprints/execute_job.py

```diff
@@ -0,0 +1,127 @@
+from httprequest_blueprints import execute_request, download_file
+import check_run_status
+import download_logs_artifacts
+import argparse
+import os
+import json
+import time
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--api-key', dest='api_key', required=True)
+    parser.add_argument('--account-id', dest='account_id', required=True)
+    parser.add_argument('--job-id', dest='job_id', required=True)
+    parser.add_argument(
+        '--download-artifacts',
+        dest='download_artifacts',
+        default='TRUE',
+        choices={
+            'TRUE',
+            'FALSE'},
+        required=False)
+    parser.add_argument(
+        '--download-logs',
+        dest='download_logs',
+        default='TRUE',
+        choices={
+            'TRUE',
+            'FALSE'},
+        required=False)
+    parser.add_argument(
+        '--execute-only',
+        dest='execute_only',
+        default='FALSE',
+        choices={
+            'TRUE',
+            'FALSE'},
+        required=False)
+    args = parser.parse_args()
+    return args
+
+
+def write_json_to_file(json_object, file_name):
+    with open(file_name, 'w') as f:
+        f.write(
+            json.dumps(
+                json_object,
+                ensure_ascii=False,
+                indent=4))
+
+
+def execute_job(
+        account_id,
+        job_id,
+        header,
+        folder_name='dbt-blueprint-logs',
+        file_name='job_details_response.json'):
+    execute_job_url = f'https://cloud.getdbt.com/api/v2/accounts/{account_id}/jobs/{job_id}/run/'
+    body = {"cause": "Run from Postman"}
```

Comment:
No longer running just from Postman. May want to consider updating to something like `print(f'Run from {os.environ["USER"]}@{os.environ["HOSTNAME"]}')`

Reply:
Or include the log id from the Shipyard environment.